### DISCUSSION TITLE
blog/2023-04-30-causal-inference-with-bayesian-models/

### DISCUSSION THREAD

**Discussion by giscus** [ASKurz/blogdown5#22] [Comments]
# blog/2023-04-30-causal-inference-with-bayesian-models/

Part 4 of the GLM and causal inference series.

https://solomonkurz.netlify.app/blog/2023-04-30-causal-inference-with-bayesian-models/

**Comment by mvuorre** [comment]
Hi Solomon, thanks so much for writing this series of blog posts. Very interesting stuff. ⭐

**Reply by ASKurz** [reply]
Thanks @mvuorre; I'm glad to hear it!
